sdk/tests/invariant.test.ts

```diff
@@ -245,6 +253,138 @@ describe('invariant', async () => {
     const pools = await invariant.getPools(account)
     assert.deepEqual(pools.length, 1)
   })
+
+  describe('positions', async () => {
+    beforeEach(async () => {
+      const feeTier = newFeeTier({ v: 10000000000n }, 1)
```

Comment:
Why not define that kind of variable inside the 'position' context? It's common for all test cases